mobile/src/package.json

```diff
@@ -14,13 +14,15 @@
     "@react-navigation/native": "^5.7.3",
     "@react-navigation/stack": "^5.9.0",
     "axios": "^0.20.0",
-    "expo": "^38.0.0",
+    "expo": "^38.0.10",
+    "expo-cli": "^3.27.10",
```

Comment:
uhhhhhhhhhhhhh I think we should undo this. We aren't supposed to add expo-cli to the local packages. It should be global instead.

Reply:
ok